ffmpeg/README.md

```diff
@@ -0,0 +1,119 @@
+# ffmpeg Compilation Guide
+The following instructions are based off of the [official ffmpeg compilation guide](https://trac.ffmpeg.org/wiki/CompilationGuide).
+If problems are encountered during this guide, please refer to the official guide.
+
+This guide sets up [Oracle's VirtualBox](https://www.virtualbox.org/wiki/Downloads) VMs to compile the ffmpeg library.
+That way:
+
+0. Most environmental inconsistencies are mitigated
```

Comment:
✏️ I suggest changing these into 1's so the list starts at 1 instead of 0
```
1. Most environmental inconsistencies are mitigated
1. Cleanup isn't needed.
```